test/integration/serverless/serverless.yml

```diff
@@ -241,28 +334,21 @@ functions:
       DD_TRACE_ENABLED: true
       DD_APM_DD_URL: http://127.0.0.1:3333
 
-  metric-csharp:
-    runtime: dotnetcore3.1
-    handler: CsharpHandlers::AwsDotnetCsharp.Handler::Hello
+  simple-trace-java:
+    runtime: java8.al2
+    handler: TraceHandler
     package:
       individually: true
-      artifact: src/csharp-tests/bin/Release/netcoreapp3.1/handler.zip
-    layers:
-      - { Ref: RecorderExtensionLambdaLayer }
-      - { Ref: DatadogExtensionIntegrationTestLambdaLayer }
-    environment:
-      DD_TAGS: tagA:valueA tagB:valueB
-      DD_EXTRA_TAGS: tagC:valueC tagD:valueD
-
-  log-csharp:
-    runtime: dotnetcore3.1
-    handler: CsharpHandlers::AwsDotnetCsharp.Handler::Logs
-    package:
-      individually: true
-      artifact: src/csharp-tests/bin/Release/netcoreapp3.1/handler.zip
+      artifact: src/java-tests/trace/target/trace-dev.jar
     layers:
+      - arn:aws:lambda:${self:provider.region}:601427279990:layer:dd-trace-java-test:${env:JAVA_TRACE_LAYER_VERSION}
```

Comment:
`601427279990` is the sandbox account, we should pull the layers from the production account for these tests.